src/main/java/cms/gov/madie/measure/services/BundleService.java

```diff
@@ -40,7 +43,27 @@ public ResponseEntity<byte[]> exportBundleMeasure(Measure measure, String access
     if (measure == null) {
       return null;
     }
-
+    if (measure.getMeasureMetaData() != null) {
+      if (CollectionUtils.isEmpty(measure.getMeasureMetaData().getDevelopers())) {
+        throw new InvalidResourceBundleStateException(
+            "Measure", measure.getId(), "since there are no associated developers in metadata.");
+      } else if (measure.getMeasureMetaData().getSteward() == null) {
+        throw new InvalidResourceBundleStateException(
+            "Measure", measure.getId(), "since there is no associated steward in metadata.");
+      } else if (measure.getMeasureMetaData().getDescription() == null) {
+        throw new InvalidResourceBundleStateException(
+            "Measure", measure.getId(), "since there is no description in metadata.");
+      }
+    }
+    if (!measure.getGroups().stream()
+        .filter(group -> group.getMeasureGroupTypes() == null)
+        .collect(Collectors.toList())
+        .isEmpty()) {
```

Comment:
```suggestion
    if (!measure.getGroups().stream().anyMatch(g -> g.getMeasureGroupTypes() != null)) {
```

Java Streams has [anyMatch](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/stream/Stream.html#anyMatch(java.util.function.Predicate)) that will return once the predicate matches.